cmd/xl-storage-disk-id-check.go

```diff
@@ -1041,28 +1041,41 @@ func (p *xlStorageDiskIDCheck) checkHealth(ctx context.Context) (err error) {
 
 // monitorDiskStatus should be called once when a drive has been marked offline.
 // Once the disk has been deemed ok, it will return to online status.
-func (p *xlStorageDiskIDCheck) monitorDiskStatus(spent time.Duration) {
+func (p *xlStorageDiskIDCheck) monitorDiskStatus(spent time.Duration, fn string) {
+	// Make sure we do not write O_DIRECT aligned I/O because WrIteAll() ends
+	// up using O_DIRECT codepath which internally utilizes p.health.tokens
+	// we need to avoid using incoming I/O tokens as part of the healthcheck
+	// monitoring I/O.
+	toWrite := []byte{2048: 42}
```

Comment:
Would it make sense making `toWrite` global if it never changes so it's not allocated over and over during monitoring?

Not sure it would help with anything though :)

Reply:
I can @kobalicek